### PR TITLE
Adding a new feature to the develop branch

### PR DESCRIPTION
**Pull request by shobancs**

### CODE REVIEW

File: src/main/java/org/springframework/samples/petclinic/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* This is a simple Java program. 
+FileName : "HelloWorld.java". */
+class HelloWorld 
+{ 
+	// Your program begins with a call to main(). 
+	// Prints "Hello, World" to the terminal window. 
+	public static void main(String args[]) 
+	{ 
+		System.out.println("Hello, World");
```
Comment (shobancs) [inline src/main/java/org/springframework/samples/petclinic/HelloWorld.java:14]: Hey, System.out.println is not accepted in the code and you need enable logging to this class

**Review by shobancs — COMMENTED** [review]
Do a change

**Review by rkolagani — COMMENTED** [review]
Please add a method

**Review by annekalyan — CHANGES_REQUESTED** [review]
Remove unwanted comments

File: src/main/java/org/springframework/samples/petclinic/HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* This is a simple Java program. 
+FileName : "HelloWorld.java". */
+class HelloWorld 
+{ 
+	
+	void wfh(){
```
Comment (shobancs) [inline src/main/java/org/springframework/samples/petclinic/HelloWorld.java:6]: wfh is not a acceptable method, choose a right naming for this method

**Review by annekalyan — APPROVED** [review]

**Review by satishgarimella — APPROVED** [review]
h

**Pull request merged**